plugin/src/test/java/org/opennms/timeseries/cortex/CortexTSSIntegrationTest.java

```diff
@@ -144,6 +144,22 @@ public void shouldGetSamplesForLongDuration() throws StorageException, Interrupt
                 timeOfLastSample, expectedEarliestTimeOfLastSample),timeOfLastSample.isAfter(expectedEarliestTimeOfLastSample));
     }
 
+    @Test
+    @Ignore
+    public void shouldFindWithNotEquals() throws Exception {
+        // Cortex doesn't seem to work with negative comparison even though the documentation says so.
+        // Maybe we query wrong?
+        // Example: http://localhost:9009/prometheus/api/v1/series?match[]={resourceId!="snmp:1:opennms-jvm:org_opennms_newts_name_ring_buffer_max_size_unit=unknown"}
```

Comment:
weird problem. I didn't spend time on it since it's not relevant for OpenNMS at the moment but it would be nice if we could fix it...

Reply:
agreed, can ignore for now